functional-tests/wdio.conf.ts

```diff
@@ -23,7 +23,7 @@ export const config: WebdriverIO.Config = {
 
 	logLevel: "error",
 
-	baseUrl: "http://localhost:3000/",
+	baseUrl: "http://localhost:4000/",
```

Comment:
Does this need to change?

Reply:
I mean can it change back to 
```suggestion
	baseUrl: "http://localhost:3000/",
```